neurons/validator/forward.py

```diff
@@ -23,6 +27,58 @@
 
 transform = T.Compose([T.PILToTensor()])
 
+def get_human_voting_scores(self):
+    max_retries = 3
+    backoff = 2
+
+    print("Querying for human votes...")
+    for attempt in range(0, max_retries):
+        try:
+
+            human_voting_scores = requests.get(f"{self.api_key}/votes", timeout=2)
+
+            if (human_voting_scores.status_code != 200) and (attempt == max_retries):
+                
+                print(f"Failed to retrieve the human validation bot votes {attempt+1} times. Skipping until the next step.")
+                return None
+
+            elif (human_voting_scores.status_code != 200) and (attempt != max_retries):
+                
+                continue
+
+            else:
+                
+                human_voting_bot_round_scores = human_voting_scores.json()
+                
+                human_voting_bot_scores = {}
+
+                for inner_dict in human_voting_bot_round_scores.values():
+                    for key, value in inner_dict.items():
+                        if key in human_voting_bot_scores:
+                            human_voting_bot_scores[key] += value
+                        else:
+                            human_voting_bot_scores[key] = value
+                
+                human_voting_bot_scores = torch.tensor([human_voting_bot_scores[key] if key in human_voting_bot_scores.keys() else 0 for key in self.hotkeys]).to(self.device)
```

Comment:
I guess it can be nice to format with `black`?

`pip install black`

Reply:
Ah good spot! I have the extension installed by default and it applies black formatting every time I save automatically. This must have been missed by that extension for some unknown reason. Just resolved.